custom_components/zwave_mqtt/cover.py

```diff
@@ -0,0 +1,136 @@
+"""Support for Z-Wave cover."""
+import logging
+
+from openzwavemqtt.const import CommandClass
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    ATTR_TILT_POSITION,
+    SUPPORT_CLOSE,
+    SUPPORT_CLOSE_TILT,
+    SUPPORT_OPEN,
+    SUPPORT_OPEN_TILT,
+    SUPPORT_SET_POSITION,
+    SUPPORT_SET_TILT_POSITION,
+    CoverDevice,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import DOMAIN, MANUFACTURER_ID_FIBARO, PRODUCT_TYPE_FIBARO_FGRM222
+from .entity import ZWaveDeviceEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORTED_FEATURES_POSITION = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+SUPPORTED_FEATURES_TILT = (
+    SUPPORT_OPEN_TILT | SUPPORT_CLOSE_TILT | SUPPORT_SET_TILT_POSITION
+)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up Z-Wave Cover from Config Entry."""
+
+    @callback
+    def async_add_cover(values):
+        """Add Z-Wave Cover."""
+        # Specific Cover Types
+        if values.primary.command_class == CommandClass.SWITCH_MULTILEVEL:
```

Comment:
If we invert this check and log warning and return if true, we can outdent below.
```suggestion
        if values.primary.command_class != CommandClass.SWITCH_MULTILEVEL:
```